site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -36,16 +30,18 @@ describe('Tests cases abut changing user pronouns', () => {
 
         //change back to old pronouns
         cy.get('#pronouns_val').click();
+
+        //set to default
         cy.get('button[aria-label="Clear pronoun input"]').click(); //clear input using trash can
-        if (oldPronouns !== '') {
-            cy.get('#user-pronouns-change').type(oldPronouns);
-        }
-        cy.get('#edit-pronouns-submit').first().click();
-
-        //ensure pronouns changed on page
-        if (oldPronouns !== '') {
-            cy.get('#pronouns_val').contains(oldPronouns);
-        }
+
+        //set to false
+        cy.get('#pronouns-forum-display').uncheck();
+
+        cy.get('#edit-pronouns-submit').click();
+
+        //ensure pronouns and display option changed on page
+        cy.get('#pronouns_val').should('have.text', ' ');
+        cy.get('#display_pronouns_val').contains('False');
```

Comment:
if you are asserting that an element contains a value, the most clear way is to do cy.get().should('contain', 'text_to_contain').

cy.get().contains('text_to_contain') will have the same output (it will fail if the element doesn't have the text), but it is for requesting the element, to then chain commands after it (like on line 78), instead of asserting that the element exists.